src/http_tracker_connection.cpp

```diff
@@ -504,7 +504,7 @@ namespace libtorrent
 			else
 #endif
 			{
-				resp.peers4.reserve(len / 6);
+				resp.peers4.reserve(std::size_t(len / 6));
```

Comment:
well, not for the PR, but it would be something very similar to ``numeric_cast<>()``, but an assert instead of a run-time check